src/forwarder.ts

```diff
@@ -0,0 +1,34 @@
+import Service from "serverless/classes/Service";
+
+const logGroupKey = "AWS::Logs::LogGroup";
+const logGroupSubscriptionKey = "AWS::Logs::SubscriptionFilter";
+
+interface LogGroupResource {
+  Type: typeof logGroupKey;
+  Properties: {
+    LogGroupName: string;
+  };
+}
+
+function isLogGroup(value: any): value is LogGroupResource {
+  return value.Type === logGroupKey;
+}
+
+export function addCloudWatchForwarderSubscriptions(service: Service, functionArn: string) {
+  const resources = service.provider.compiledCloudFormationTemplate.Resources;
+
+  for (const [name, resource] of Object.entries(resources)) {
+    if (!isLogGroup(resource) || !resource.Properties.LogGroupName.startsWith("/aws/lambda/")) {
```

Comment:
nits, does it make the code more explicit by moving the `Properties.LogGroupName.startsWith("/aws/lambda/")` check into the helper function and rename the helper function to `isLambdaLogGroup`?

Reply:
The other function is a type guard, so it's just being used to confirm the structure of the object is correct. Since the '/aws/lambda' isn't really encoded into the interface/type, my thinking was it shouldn't live inside the type guard.